MVPByCode/Delegation/DelegationView.swift

```diff
@@ -0,0 +1,197 @@
+        stackView.distribution  = .equalSpacing
+        stackView.spacing = 24
+        return stackView
+    }()
+    
+    var secondStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis  = .horizontal
+        stackView.distribution  = .fillProportionally
+        stackView.spacing = 8
+        return stackView
+    }()
+    
+    var nametxtField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "First name"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+        
+    }()
+    
+    
+    var surnametxtField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "Last name"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+        
+    }()
+    
+    var emailtxtField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "Email"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+        
+    }()
+    
+    var cardtxtField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "Carta d'identità"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+        
+    }()
+    
+    var prefixTextField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "Prefix"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    var phoneTxtField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.placeholder = "Phone"
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    var sendButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .black
+        button.setTitle("Send", for: .normal)
+        button.tintColor = .white
+        return button
+    }()
+    
+    init() {
+        super.init(frame: .zero)
+        backgroundColor = .white
+        configureUI()
+        configureConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configureUI() {
+        self.addSubview(scrollView)
+        scrollView.addSubview(contentView)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(descriprionLabel)
+        contentView.addSubview(nametxtField)
+        contentView.addSubview(surnametxtField)
+        contentView.addSubview(cardtxtField)
+        contentView.addSubview(emailtxtField)
+        contentView.addSubview(mainStackView)
+        mainStackView.addArrangedSubview(nametxtField)
```

Comment:
Meglio usare il costruttore della stackView ed inizializzalo direttamente con le subviews anziché aggiungerle in un secondo tempo.